src/main/java/at/fhhagenberg/model/Building.java

```diff
@@ -0,0 +1,168 @@
+package at.fhhagenberg.model;
+
+import java.util.Comparator;
+import java.util.List;
+
+// contains information about all elevators of a building
+public class Building {
+    private final List<Elevator> mElevators;
+
+    private final List<Floor> mFloors;
+
+    private final boolean[][] mElevatorToFloorServiceMapping;
```

Comment:
kommentar wäre nice gewesen. bin ma ned sicher was des macht

Reply:
hab mi scho erinnert was des is. sorry for the inconvenience. have a nice day god bless you bye